### PR TITLE
[IE VPU] Execution graph via ngraph for VPU plugin

### PR DESCRIPTION
**Pull request by adudchen**
#-33036

### CODE REVIEW

File: inference-engine/src/vpu/graph_transformer/src/utils/runtime_graph.cpp
Comment (DariaMityagina) [inline inference-engine/src/vpu/graph_transformer/src/utils/runtime_graph.cpp:7]: Not really important but `ie_ngraph_utils.hpp` includes `string`, so maybe line 16 is redundant now.

File: inference-engine/tests/functional/plugin/shared/include/behavior/exec_graph_info.hpp
Comment (ilya-lavrenov) [inline inference-engine/tests/functional/plugin/shared/include/behavior/exec_graph_info.hpp:231]: could you please just remote this `if`? the body of `if` should be the same for all devices.
BTW, don't we need to enable this tests for HDDL as well?

File: inference-engine/src/vpu/graph_transformer/src/utils/runtime_graph.cpp
Comment (ilya-lavrenov) [inline inference-engine/src/vpu/graph_transformer/src/utils/runtime_graph.cpp:8]: why do we need `inference_engine.hpp`?

**Comment by ilya-lavrenov** [comment]
@adudchen let's finalize by 2021.1

File: inference-engine/tests/functional/plugin/shared/include/behavior/exec_graph_info.hpp
Comment (adudchen) [reply inference-engine/tests/functional/plugin/shared/include/behavior/exec_graph_info.hpp]: I'll enable these tests for the HDDL plugin after merging this PR.

**Comment by ilya-lavrenov** [comment]
@adudchen please, merge with latest master

**Comment by ilya-lavrenov** [comment]
@adudchen please, have a look https://openvino-ci.intel.com/job/private-ci/job/ie/job/ie-tests-linux-ubuntu18-gna/12296/

File: inference-engine/src/vpu/graph_transformer/src/utils/runtime_graph.cpp
Comment (adudchen) [reply inference-engine/src/vpu/graph_transformer/src/utils/runtime_graph.cpp]: Fixed

**Review by Maxim-Doronin — APPROVED** [review]
Some style issues. But in general LGTM

**Comment by Maxim-Doronin** [comment]
@ilya-lavrenov, can we merge this PR right now?

**Comment by ilya-lavrenov** [comment]
@Maxim-Doronin I think yes, since it's passed review from VPU maintainers

**Pull request merged**
